error_plan9.go

```diff
@@ -0,0 +1,10 @@
+// +build plan9
+
+package hdfs
+
+import (
+	"errors"
+)
+
+// plan9 does not have ENOTEMPTY
+var osErrNotEmpty = errors.New(pathIsNotEmptyDirException)
```

Comment:
This swallows the error. You can just reimplement the switch statement in this file.

Reply:
Yes. It's fine with me. I could not find an equivalent for plan9 so I assume it is not diagnosed on the OS level.